packages/flutter/pubspec.yaml

```diff
@@ -36,3 +36,4 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: ^5.2.0
+  path_provider_platform_interface: ^2.0.6
```

Comment:
Welcome to the Parse Flutter SDK review team ❤️
and i tested the code with this package with the following command
```dart
flutter pub publish --dry-run
```
One of the errors has been solved, but the other error remains, I suggest you add the following package
```
plugin_platform_interface: ^2.1.4
```

Reply:
it's working now.